documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,444 @@
++++
+title = "Common Mounts"
+date = 2019-02-23T16:45:16-05:00
+weight = 70
+pre = "<b> </b>"
++++
+
+#### Contents
+
+ - [Introduction](#introduction)
+ - [Configuration](#configuration)
+ - [Common Mounts Fields](#common-mounts-fields)
+ - [Common Mount Volumes Fields](#common-mount-volumes-fields)
+ - [Running Model in Image initial use case using common mounts](#running-model-in-image-initial-use-case-using-common-mounts)
+    - [Creating the common mounts image](#creating-the-common-mounts-image)
+    - [Domain resource](#domain-resource)
+
+### Introduction
+Use common mounts to automatically include the directory content from additional images. This is a useful alternative for including Model in Image model files, or other types of files, in a Pod without requiring modifications to the Pod's base image specified using `domain.spec.image`. This feature internally uses a Kubernetes [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume and Kubernetes [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to share the files from the additional images with the Pod.
+
+When using the common mounts feature with the Model In Image use case, the WebLogic image would be the Domain resource's `domain.spec.image` field. The model files, application archives, and the WDT installation files can be provided in a separate image specified as part of the common mounts. The application archive could also be provided by a second container image in the common mounts section. Specific advantages of the common mounts feature for Model In Image domains are:
+  - Use a WebLogic install image, or patch a WebLogic install image, without needing to include model artifacts within the image.
+  - Share one large WebLogic install image with multiple different model configurations.
+  - Distribute model files, application archives, and WebLogic Deploy Tooling executable using very small images instead of a large image that also contains a WebLogic install.
```

Comment:
Proposed rewording:

Common mounts are an alternative approach for including Model in Image model files, application archive files, Weblogic Deploying Tooling install files, or other types of files, in your pods. This feature eliminates the need to provide these files in the image specified in `domain.spec.image`.  Instead:
- The domain resource's `domain.spec.image` directly reference a base image that only needs to include a WebLogic install and a Java install
- The domain resource's common mount related fields reference one or more smaller images that contain the desired Model in Image files.
- The domain resource `domain.spec.configuration.model.wdtInstallHome` and `domain.spec.configuration.model.modelHome` fields ares set to reference a directory that contains the files from the smaller images.

The common mounts feature internally uses a Kubernetes [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume and Kubernetes [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to share files from additional images within a WebLogic Server pod or the Introspector job pod.

The advantages of the common mounts feature for Model In Image domains are:  
- Use or patch a WebLogic install image without needing to include WDT install, application archive, or model artifacts within the image.  
- Share one large WebLogic install image with multiple different model configurations that are supplied in smaller images. 
- Distribute or update model files, application archives, and the WebLogic Deploy Tooling executable using very small images instead of a large image that also contains a WebLogic install.